client/src/templates/Challenges/components/completion-modal.tsx

```diff
@@ -92,6 +92,22 @@ export function getCompletedPercent(
   return completedPercent > 100 ? 100 : completedPercent;
 }
 
+export function getCompletedProjects(
+  completedChallengesIds: string[] = [],
+  currentBlockIds: string[] = [],
+  currentChallengeId: string
+): string {
+  completedChallengesIds = completedChallengesIds.includes(currentChallengeId)
+    ? completedChallengesIds
+    : [...completedChallengesIds, currentChallengeId];
+
+  const completedChallengesInBlock = completedChallengesIds.filter(id => {
+    return currentBlockIds.includes(id);
+  });
+
+  return `${completedChallengesInBlock.length} of ${currentBlockIds.length}`;
```

Comment:
We cannot use a string including english words as part of a translation. Also, it makes more sense for this to be either a `{numerator, denominator}` or `"<NUMERATOR>/<DENOMINATOR>"`

Reply:
Hey @ShaunSHamilton 

If,

>We cannot use a string including english words as part of a translation.

then what shall I use for variable names?

I think I am missing some thing here.